compiler/plugins/input/StableHLO/PluginRegistration.cpp

```diff
@@ -115,6 +117,7 @@ struct StableHLOSession
   void populateCustomInputConversionTypes(StringSet<> &typeMnemonics) override {
     typeMnemonics.insert("stablehlo");
     typeMnemonics.insert("stablehlo_xla");
+    typeMnemonics.insert("vhlo");
```

Comment:
Is this still needed after the CLI option was removed?

Reply:
I think having an alias may be okay, even if the code paths underlying both are exactly the same. Then users could pass `--iree-input-type=vhlo` _or_ `--iree-input-type=stablehlo`. However, most users should be using the auto pipeline (leaving off `--iree-input-type`) anyways IMO.

Reply:
I didn't add an extra else case for VHLO, just made it part of stablehlo.